arangod/Replication/DatabaseInitialSyncer.cpp

```diff
@@ -1410,10 +1411,46 @@ Result DatabaseInitialSyncer::handleCollectionsAndViews(VPackSlice const& collSl
       }
     }
 
-    collections.emplace_back(parameters, indexes);
+    if (masterName == StaticStrings::AnalyzersCollection) {
+      // _analyzers collection has to be restored before view creation
+      systemCollections.emplace_back(parameters, indexes);
+    } else {
+      collections.emplace_back(parameters, indexes);
+    }
+  }
+
+  // STEP 1: validate collection declarations from master
+  // ----------------------------------------------------------------------------------
+
+  // STEP 2: drop and re-create collections locally if they are also present on
+  // the master
+  //  ------------------------------------------------------------------------------------
+
+  // iterate over all collections from the master...
+  std::array<SyncPhase, 2> phases{{PHASE_VALIDATE, PHASE_DROP_CREATE}};
+  for (auto const& phase : phases) {
+    Result r = iterateCollections(systemCollections, incremental, phase);
```

Comment:
Will this work in a cluster, if `_analyzers` is distributed like another system collection, such as `_users` or `_graphs`?
I think the prototype collection used for `_analyzers` must be restored before `_analyzers`.

Reply:
Ok, we probably don't use the `DatabaseInitialSyncer` in a cluster at all, so this may be a false alert of mine.